frontend/src/lib/codemirror/useCompareExtension.worker.ts

```diff
@@ -1,14 +1,16 @@
 import { diff } from "fast-myers-diff"
 
-export function calculateDiff(target: string | undefined, current: string): [number, number, number, number][] {
-    if (typeof target !== "string") {
+const ctx: Worker = self as any
+
+ctx.onmessage = ({ data }: {data: {target: string | undefined, current: string }}) => {
```

Comment:
Can you export the `{data: {target: string | undefined, current: string }}` type so it can be `import { type Message } from "./useCompareExtension.worker.ts"`'d? Just so we have some semblance of type safety

Reply:
👍